### PR TITLE
Newsletter 13: A little section about rspirv-reflect

### PR DESCRIPTION
**Pull request by Jasper-Bekkers**
As mentioned in #232, I wrote a small section on rspirv-reflect.

### CODE REVIEW

**Review by 17cupsofcoffee — COMMENTED** [review]
The general content of this PR looks good, but according to the tracking issue it's generally preferred to write in third person (i.e "Traverse Research have created rspirv-reflect"), to keep a consistent tone between sections. Could you tweak this to match that guideline? Happy to approve this other than that one little nitpick :)

**Comment by Jasper-Bekkers** [comment]
Thanks for the feedback @17cupsofcoffee, does conform to the guidelines better?

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -66,6 +66,32 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+[Traverse Research] has created the [rspirv-reflect] library to replace
+our very basic use-case of the existing [spirv-reflect-rs] / [spirv-reflect]
+libraries that are already out there. The current iteration of `rspirv-reflect`
+is pretty minimal, but it allows you to extract the binding setup from a SPIR-V
+binary. `rspirv-reflect` supports the latest version of SPIR-V (version 1.5 as
+of writing) and it also supports all the new shader stages (both ray tracing
+and mesh/task shaders) as well as the existing ones.
+
+Traverse Research  wanted to reduce their reliance on C and C++ unsafe
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:81]:
```suggestion
Traverse Research wanted to reduce their reliance on C and C++ unsafe
```

File: content/posts/newsletter-013/index.md [hunk 2]
```diff
@@ -66,6 +66,32 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+[Traverse Research] has created the [rspirv-reflect] library to replace
+our very basic use-case of the existing [spirv-reflect-rs] / [spirv-reflect]
+libraries that are already out there. The current iteration of `rspirv-reflect`
+is pretty minimal, but it allows you to extract the binding setup from a SPIR-V
+binary. `rspirv-reflect` supports the latest version of SPIR-V (version 1.5 as
+of writing) and it also supports all the new shader stages (both ray tracing
+and mesh/task shaders) as well as the existing ones.
+
+Traverse Research  wanted to reduce their reliance on C and C++ unsafe 
+libraries and at the same time we needed to support newer features that were
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:82]:
```suggestion
libraries and at the same time we needed to support newer features that were
```

File: content/posts/newsletter-013/index.md [hunk 3]
```diff
@@ -66,6 +66,32 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+our very basic use-case of the existing [spirv-reflect-rs] / [spirv-reflect]
+libraries that are already out there. The current iteration of `rspirv-reflect`
+is pretty minimal, but it allows you to extract the binding setup from a SPIR-V
+binary. `rspirv-reflect` supports the latest version of SPIR-V (version 1.5 as
+of writing) and it also supports all the new shader stages (both ray tracing
+and mesh/task shaders) as well as the existing ones.
+
+Traverse Research  wanted to reduce their reliance on C and C++ unsafe 
+libraries and at the same time we needed to support newer features that were 
+slow to become available in the existing `spirv-reflect` library. The primary
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:83]:
```suggestion
slow to become available in the existing `spirv-reflect` library. The primary
```

File: content/posts/newsletter-013/index.md [hunk 4]
```diff
@@ -66,6 +66,32 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+libraries that are already out there. The current iteration of `rspirv-reflect`
+is pretty minimal, but it allows you to extract the binding setup from a SPIR-V
+binary. `rspirv-reflect` supports the latest version of SPIR-V (version 1.5 as
+of writing) and it also supports all the new shader stages (both ray tracing
+and mesh/task shaders) as well as the existing ones.
+
+Traverse Research  wanted to reduce their reliance on C and C++ unsafe 
+libraries and at the same time we needed to support newer features that were 
+slow to become available in the existing `spirv-reflect` library. The primary 
+use-case for this library is in conjecture with the Rust wrapper around the
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:84]:
```suggestion
use-case for this library is in conjecture with the Rust wrapper around the
```

File: content/posts/newsletter-013/index.md [hunk 5]
```diff
@@ -66,6 +66,32 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+is pretty minimal, but it allows you to extract the binding setup from a SPIR-V
+binary. `rspirv-reflect` supports the latest version of SPIR-V (version 1.5 as
+of writing) and it also supports all the new shader stages (both ray tracing
+and mesh/task shaders) as well as the existing ones.
+
+Traverse Research  wanted to reduce their reliance on C and C++ unsafe 
+libraries and at the same time we needed to support newer features that were 
+slow to become available in the existing `spirv-reflect` library. The primary 
+use-case for this library is in conjecture with the Rust wrapper around the 
+DirectX Shader Compiler ([dxc]), called [hassle-rs] that Traverse Research
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:85]:
```suggestion
DirectX Shader Compiler ([dxc]), called [hassle-rs] that Traverse Research
```

**Review by 17cupsofcoffee — APPROVED** [review]
Yeah, looks better now :+1: Just a few more tweaks needed (mainly just to make the CI build happy), but will approve in advance.

**Comment by ozkriff** [comment]
@Jasper-Bekkers As I don't have write permission to this PR, I made a meta PR with required style tweaks: https://github.com/Traverse-Research/rust-gamedev.github.io/pull/1

Could you also resolve the merge conflicts, please?

**Comment by ozkriff** [comment]
Merged manually (https://github.com/rust-gamedev/rust-gamedev.github.io/commit/a68fb5f7ff655eb5b97911cc688c827001af9740) as I don't have write permission to the PR's branch.

**Pull request closed without merging**